templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete driver?</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+
+    <input type="submit" value="Yes" class="btn btn-danger">
+  </form>
+{% endblock %}
```

Comment:
![Screenshot from 2023-10-22 12-08-54](https://github.com/mate-academy/py-taxi-service-user-form-and-validation/assets/122517203/6e288c5c-b846-49b8-b9d2-20e9caa348ad)
Read checklist